WordPress/src/main/java/org/wordpress/android/ui/stats/StatsCommentsFragment.java

```diff
@@ -82,7 +82,7 @@ protected void updateUI() {
             return;
         }
 
-        if (mDatamodels[1] != null) { // check if comment-followers is already here
+        if (mDatamodels[1] != null && !isErrorResponse(1)) { // check if comment-followers is already here
```

Comment:
Sorry, I was not clear. The object received is either a CommentFollowersModel or a Error Type.